crates/iota-indexer/migrations/pg/2025-10-01-090835_watermarks/up.sql

```diff
@@ -1,29 +1,34 @@
-CREATE TABLE watermarks
+CREATE TABLE IF NOT EXISTS watermarks
 (
-    -- The table governed by this watermark, i.e `epochs`, `checkpoints`, `transactions`.
-    entity                      TEXT          NOT NULL,
-    -- Inclusive upper epoch bound for this entity's data. Committer updates this field. Pruner uses
-    -- this to determine if pruning is necessary based on the retention policy.
+    -- The pipeline governed by this watermark, i.e `epochs`, `checkpoints`,
+    -- `transactions`.
+    pipeline                    TEXT          PRIMARY KEY,
```

Comment:
Doesn't this imply that each `PrunableTable` has its own pipeline? I think `entity` makes more sense.